src/gluonts/mx/trainer/_base.py

```diff
@@ -40,6 +40,13 @@
     save_epoch_info,
 )
 
+# iteration averaging
```

Comment:
I guess for consistency we should remove the `# iteration averaging` comment since this falls under the relative imports.